lcs/agents/acs2/ACS2.py

```diff
@@ -130,6 +130,7 @@ def _run_trial_explore(self, env, time, current_trial=None) \
 
             steps += 1
 
+        # Reward is not accumulated; only last step reward is returned
         return TrialMetrics(steps, reward)
```

Comment:
Yes, good point. It's a little bit vague here.
Could you rename the variable `reward` to `last_reward` instead of adding a comment?

Reply:
Sure, will do.